.github/workflows/pipeline-build.yml

```diff
@@ -26,11 +27,38 @@ jobs:
           uses: better0fdead/actions-setup-docker@better0fdead/increase-timeout
 
         #Build without test on Windows or macOS (when not a PR)
```

Comment:
@SaeedRe this comment refers to line 58, so it should be moved there or thereabouts.